.erdconfig

```diff
@@ -1,4 +1,4 @@
-title: login.gov IdP entity-relationship diagram
```

Comment:
Aside: @jmhooper should this file have been removed in #4960? I never noticed it before.

Reply:
Yeah, this should have been removed

Reply:
Cleanup: https://github.com/18F/identity-idp/pull/5226